exercises/C-comparison-operators/exercise.js

```diff
@@ -7,29 +7,29 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = true; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = true; // finish this statement
 
 let personA = "Daniel";
 let personB = "Irina";
-let sameName; // finish this statement
+let sameName = false; // finish this statement
```

Comment:
Hi Anderson. Great job! You can also use comparison operators (=== same value) like this;

let sameName = personA === personB;